apps/jobs/views.py

```diff
@@ -427,6 +428,54 @@ def get_job_categories(self, request):
                 serialized_job_categories, status.HTTP_200_OK
             )
 
+    @action(detail=False, methods=["get"])
+    def get_jobs(self, request):
```

Comment:
As per the FE requirement's updated the code. Added filters which can dynamically retrieve the data according to the request.

1) Retrive jobs as per quary parameter.
```
request: http://localhost:8000/jobs/get_jobs/?category=Security&job_type=full time&qualifications=OWASP&experience=0

response: {
            "job_id": "4332fa40-2c96-43a7-9eae-6978075a3be3",
            "job_role": "Security Analysist",
            "location": "Mumbai",
            "post_date": "2023-10-11",
            "posted": true,
            "experience": 0,
            "created_at": "2024-01-11T18:04:30.138365Z",
            "updated_at": "2024-01-11T18:04:30.138365Z",
            "employer_id": "5e1f2925-9deb-492f-8a06-f451daf9310a",
            "job_type": "full time",
            "salary": "30000.00",
            "qualifications": "OWASP",
            ..... 
          }
```
2) Dynamically retriving of jobs as per the queary updation
```
request: http://localhost:8000/jobs/get_jobs/?job_type=full time&experience=0
response: {
            "job_id": "be5f509a-855a-4de8-b13e-7cb6cf2243fc",
            "job_role": "Product Manager",
            "location": "California",
            "post_date": "2023-10-11",
            "posted": true,
            "experience": 0,
            "created_at": "2024-01-11T17:58:36.746918Z",
            "updated_at": "2024-01-11T17:58:36.746918Z",
            "employer_id": "5e1f2925-9deb-492f-8a06-f451daf9310a",
            "job_type": "full time",
            "salary": "20000.00",
            "qualifications": "Commerces, Data Managment",
            ...
```